wooey/management/commands/addscript.py

```diff
@@ -26,15 +40,50 @@ def handle(self, *args, **options):
                 script = args[0]
             else:
                 raise CommandError('You must provide a script path or directory containing scripts.')
+
```

Comment:
It might be useful to see how Django uses the --template parameter for their project bootstrapper to see if we are missing any corner cases.